sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationClient.cs

```diff
@@ -430,7 +427,7 @@ public virtual Response<IReadOnlyList<FileFormat>> GetDocumentFormats(Cancellati
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         public virtual async Task<Response<IReadOnlyList<FileFormat>>> GetDocumentFormatsAsync(CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(DocumentTranslationClient)}.{nameof(GetDocumentFormatsAsync)}");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(DocumentTranslationClient)}.{nameof(GetDocumentFormats)}");
```

Comment:
our `ClientCannotAuthenticateWithFakeApiKey` test actually caught this bug.
Yay for tests!!!